CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 - Introduce Entrypoint and Parameter types, and their owned versions.
 - Add a new schema version for V1 smart contracts. 
   This adds schema for return values of init and receive functions, and removes the state schema.
+- Implement `quickcheck::arbitrary` for `Timestamp`, `AccountAddress`, `ContractAddress`, `Address`,
+  `ChainMetadata`, `AttributeTag`, and `OwnedPolicy`. 
```

Comment:
This should go under the Unreleased changes section.

Reply:
Done